roles/edpm_libvirt/tasks/install.yml

```diff
@@ -21,6 +21,11 @@
     enabled: false
     masked: true
     state: stopped
+  loop:
```

Comment:
line 20 should be `name: "{{ item }}"`

Reply:
... yes

so they are disabeld and stopped by default but not masked

which means i also am missing a verify test fo the sockets.

ill respin this tomorrow thanks